### PR TITLE
doc: cpu-sharing doc update

### PR DESCRIPTION
**Pull request by conghuic23**
Update cpu-sharing document.

Signed-off-by: Conghui Chen <conghui.chen@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: doc/tutorials/cpu_sharing.rst [hunk 1]
```diff
@@ -37,21 +37,29 @@ Scheduling initialization is invoked in the hardware management layer.
...
-``--cpu_affinity``. Use these rules to configure the vCPU affinity:
-
-- Only one bit can be set for each affinity item of vCPU.
-- vCPUs in the same VM cannot be assigned to the same pCPU.
+Currently, we do not support vCPU migration; the assignment of vCPU mapping to
+pCPU is fixed at the time the VM is launched. The statically configured
+cpu_affinity in the VM configuration defines a superset of pCPUs that
+the VM is allowed to run on. One bit in this bitmap indicates that one pCPU
+could be assigned to this VM, and the bit number is the pCPU ID. A pre-launched
+VM is supposed to be launched on exact number of pCPUs that are assigned in
```
Comment (gvancuts) [inline doc/tutorials/cpu_sharing.rst:48]: ... on the exact...?

Comment (conghuic23) [reply doc/tutorials/cpu_sharing.rst:48]: Yes, but "cpu_affinity" in VM configuration is for master, "cpu_affinity_bitmap" is for release_1.6.1.

**Pull request merged**